app/javascript/components/mentoring/mentor_solutions_list/solution_list.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useReducer, useCallback } from 'react'
```

Comment:
`useReducer` and `useCallback` are unused:

```suggestion
import React from 'react'
```